lightly_studio/src/lightly_studio/few_shot_classifier/classifier_manager.py

```diff
@@ -465,9 +466,9 @@ def run_classifier(self, session: Session, classifier_id: UUID, dataset_id: UUID
         predictions = classifier.few_shot_classifier.predict(embeddings)
         if len(predictions):
             _create_annotation_labels_for_classifier(
-                classifier=classifier,
                 session=session,
                 dataset_id=dataset_id,
+                classifier=classifier,
```

Comment:
In the progress on making this PR, I had a change here that later I removed. I reordered the parameters to match the function order.